marathon_lb.py

```diff
@@ -1307,7 +1307,7 @@ def get_health_check(app, portIndex):
 healthCheckResultCache = LRUCache()
 
 
-def get_apps(marathon, apps=[]):
+def get_apps(marathon, apps=[], groups=False):
```

Comment:
I think its better to define function as `get_apps(marathon, apps=[], groups=None):`